tools/Google.Cloud.Tools.ReleaseManager/ContainerCommands/BuildCommand.cs

```diff
@@ -74,16 +75,35 @@ private int BuildConfigured(Dictionary<string, string> options, string repoRoot)
             var targetApi = catalog.Apis.SingleOrDefault(api => api.ProtoPath == apiPath);
             if (targetApi is null)
             {
-                Console.WriteLine($"API path '{apiPath}' is not configured for any API.");
-                return 1;
+                // Maybe what was passed in wasn't a proto path, but a library ID (as per the pipeline-state.json, so it may be a package group ID)...
```

Comment:
It may be a single library ID as well, right? Although those are likely to be specified through the proto-path instead?

Reply:
When Librarian is running generation - which may be "raw" (e.g. in CI, with no context of google-cloud-dotnet) it wants to build what has just been generated, and there may be no library. When Librarian is creating a release PR, it wants to be able to build the library that it's preparing to release.

It may well be that we actually want to separate the two container build commands - but for now, this feels like a quick way of reusing the existing code while we're exploring the release process.